compiler/noirc_evaluator/src/ssa/opt/loop_invariant.rs

```diff
@@ -40,7 +40,7 @@
 //! ```
 //! Reversing this for post-dominance we can see that the conditions for control dependence
 //! are the same as those for post-dominance frontiers.
-//! Thus, we rewrite our control dependence condition as Y is control dependent on X iff Y is in PDF(Y).
+//! Thus, we rewrite our control dependence condition as Y is control dependent on X iff X is in PDF(Y).
```

Comment:
This comment was incorrect. However, the code is correct. Just noting this change for the reviewer.